RHDEVDOCS-3589: understanding odo (#42313)
openshift/openshift-docs: modules/odo-telemetry.adoc

// Module included in the following assemblies:
//
// * cli_reference/developer_cli_odo/understanding-odo.adoc

:_content-type: CONCEPT
[id="odo-telemetry_{context}"]

= Telemetry in odo

> `{odo-title}` collects information about how `{odo-title}` is being  used, including metrics on operating system, RAM, CPU size, number of cores, version of `{odo-title}`, errors, success/failure, and time it takes for odo commands to complete.

Reviewer: ```suggestion
`odo` collects information about how it is being used, including metrics on the operating system, RAM, CPU, number of cores, `odo` version, errors, success/failure, and how long `odo` commands takes to complete.
```

Author: Agreed
